README.md

```diff
@@ -44,6 +55,14 @@ Alternatively, you can <a href="https://github.com/watermelontools/wm-extension#
 
 You may also highlight and right click on the code, you will find the 'Get Pull Requests with Watermelon' command at the end.
 
+### Daily Summary
+
+Get an overview of which issues and PR reviews are assigned to you. Both on the current repository and across all GitHub. 
+
+To use it, open the Watermelon sidebar.
+
+![daily-summary](https://i.ibb.co/RCym1hF/Screen-Shot-2022-07-13-at-5-03-48-PM.png)
```

Comment:
Shouldn't these media files live in [/images](https://github.com/watermelontools/wm-extension/tree/dev/media)? Or our webpage?
If we have it at our web page we may edit it without a PR

Reply:
I'm on the Webflow dashboard and don't see a straightforward way to do it. Will place it on /images

Reply:
Thank you.